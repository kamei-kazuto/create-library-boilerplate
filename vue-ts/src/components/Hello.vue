<template>
    <div class="greeting">Hello World</div>
</template>

<script lang="ts">
export default class Hello {

}
</script>

<style>
</style>
