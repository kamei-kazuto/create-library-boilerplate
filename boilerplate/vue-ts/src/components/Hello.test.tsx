import { mount } from '@vue/test-utils'
import Hello from './Hello.vue'

describe('Hello.vue', () => {
  it('render', () => {
    const wrapper = mount(Hello)
    expect(wrapper.element).toMatchSnapshot()
  })
})
