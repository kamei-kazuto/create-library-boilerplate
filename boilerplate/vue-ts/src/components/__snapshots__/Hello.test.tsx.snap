// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello.vue render 1`] = `
<div
  class="greeting"
>
  Hello World
</div>
`;
